blackberggroup/VA-OAM
# feat: add feature image to detail page; style adjustments

## What type of PR is this? (check all applicable)


## Description

- Added featured image to news article detail page 
- Additional logic checks for broken/missing data
- Updated article gallery logic

## Related Tickets & Documents
- Monday Task #6627643780

## QA Instructions, Screenshots, Recordings

-Visit news article detail page and [verify featured image is being display](https://dev-innovation.bgsandbox.com/oam/news-and-events/article/index.html?article=enhancing-surgical-outcomes-the-glenohumeral-model)

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Are there any post deployment tasks we need to perform?
- Add GA code on production
- Proper meta tags utilized

## Changed Files in this Pull Request ##
- _includes/news-and-events/newsletter-articles-dynamic.html
 - _site/assets/css/styles.css
 - _site/assets/js/news-article.js
 - _site/assets/js/news-functions.js
 - _site/assets/js/news-json-render-dynamic.js
 - _site/assets/sass/vha/pages/news/_index.scss
 - _site/news-and-events/article/index.html
 - _site/news-and-events/index.html
 - assets/css/styles.css
 - assets/js/news-article.js
 - assets/js/news-functions.js
 - assets/js/news-json-render-dynamic.js
 - assets/sass/vha/pages/news/_index.scss
 - news-and-events/article/index.html
